ironfish/src/rpc/routes/chain/utils.ts

```diff
@@ -165,6 +166,20 @@ export async function renderGraph(
   }
 }
 
+export const serializeRpcBlock = (block: Block, serialized?: boolean): RpcBlock => {
+  const blockHeaderResponse = serializeRpcBlockHeader(block.header)
+
+  const transactions: RpcTransaction[] = []
+  for (const tx of block.transactions) {
+    transactions.push(serializeRpcTransaction(tx, serialized))
+  }
+
+  return {
+    ...blockHeaderResponse,
+    size: getBlockSize(block),
+    transactions,
+  }
+}
```

Comment:
Newline here might be nice:

```suggestion
}

```